extensions/sql-migration/src/api/utils.ts

```diff
@@ -279,16 +279,15 @@ export function getMigrationStatusWithErrors(migration: azure.DatabaseMigration)
 	warningCount += properties.migrationFailureError?.message?.length > 0 ? 1 : 0;
 
 	// file upload blocking errors
-	warningCount += properties.migrationStatusDetails?.fileUploadBlockingErrors?.length ?? 0;
+	warningCount += properties.migrationStatusWarnings?.fileUploadBlockingErrorCount ?? 0;
```

Comment:
when will is this property available?  is it deployed worldwide now?

Reply:
for compatibility, will migrationStatusDetails still be returned for older migrations?

Reply:
yes, this was part of the 9/22 snap and should be deployed worldwide now. 

and yes, according to Austin's PR where he made the RP change, `MigrationStatusDetails` is still being populated as it was before (in one iteration it was removed, but then added back for backwards compatibility)